src/NHSD.BuyingCatalogue.Ordering.Api/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
```

Comment:
Would it be possible to remove these 2 lines? 

In my opinion they don't add any value to the Dockerfile itself.